toolbox/src/main/java/com/asf/appcoins/toolbox/Application.java

```diff
@@ -1,25 +1,23 @@
 package com.asf.appcoins.toolbox;
 
-import android.content.pm.PackageManager;
-
-import com.asf.appcoins.sdk.ads.AppCoinsAds;
-import com.asf.appcoins.sdk.ads.AppCoinsAdsBuilder;
 
 public class Application extends android.app.Application {
 
     public final static String developerAddress = "0x75e8BdCf84B9bd585753dB64A8637A52Bd46f851";
 
-    private static AppCoinsAds adsSdk;
+    //private static AppCoinsAds adsSdk;
```

Comment:
Check with the Wallet team how to deal with the POA being present on the toolbox